bin/reset.recovery_download

```diff
@@ -20,13 +20,21 @@
 
 . "${bin_dir}/recovery.${recovery_impl}"
 
-# USB enumeration delay
-for ((i = 0; i <= 100; i++)); do
-    if [ -e "${board_usb_dev}" ]; then
-        break
+if [[ -n "${board_usb_dev}" ]]; then
+    # USB enumeration delay
+    for ((i = 0; i <= 100; i++)); do
+        if [ -e "${board_usb_dev}" ]; then
+            break
+        fi
+        sleep 0.1
+    done
+    if [ ! -e "${board_usb_dev}" ]; then
+        echo "Cannot find device ${board_usb_dev}"
+        exit 1
     fi
-    sleep 0.1
-done
-sleep 1
+    sleep 1
+else
+    sleep 2
+fi
```

Comment:
I don't understand the need to add "sleep 1" or "sleep 2" at the end here.

Reply:
Well sleep 1 is what was there before
sleep 2 is my hack to give it long enough to see the tegra device on USB

Reply:
Ah right, I had not noticed the `-sleep 1` line in the diff; the sleep 1 makes sense in this patch, although I'm not sure why it was there before. From experience with writing code that waits e.g. for USB Mass Storage devices to appear, I doubt `sleep 2` is long enough, especially if something else just happens to run at the same time and bog the machine down.

Reply:
OK thanks. If I can't get this running I suppose I can increase it to 3.